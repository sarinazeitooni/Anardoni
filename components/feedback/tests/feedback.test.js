import React from 'react';
import { shallow } from 'enzyme';
import FeedBack from "../feedback";
describe("feedback component", () => {
    it("should have a specific text", () => {
        const feedback = shallow(<FeedBack/>);
        expect(feedback.length).toBe(1);
        const text = feedback.find('span');
        expect(text.text('بازخورد'));
        expect(feedback).toMatchSnapshot();
    });
});