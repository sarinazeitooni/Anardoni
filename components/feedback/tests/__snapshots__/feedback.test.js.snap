// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`feedback component should have a specific text 1`] = `ShallowWrapper {}`;
